From 4e0a6cf0ef9597763a1aaf1ea7e37bba8976cbf5 Mon Sep 17 00:00:00 2001
From: Clif Liu <clif.liu@lumotive.com>
Date: Thu, 11 May 2023 14:43:47 -0700
Subject: [PATCH] DTB fixes

Remove rx-internal-delay from fec side
Set i2c0 speed to 1Mbps and remove GPIO expander
Set i2c0_mipi0 speed to 1Mbps
---
 arch/arm64/boot/dts/freescale/imx8qm-mek.dts | 7 +++----
 1 file changed, 3 insertions(+), 4 deletions(-)

diff --git a/arch/arm64/boot/dts/freescale/imx8qm-mek.dts b/arch/arm64/boot/dts/freescale/imx8qm-mek.dts
index 87ebea1d6eac..2aa42040fdb2 100755
--- a/arch/arm64/boot/dts/freescale/imx8qm-mek.dts
+++ b/arch/arm64/boot/dts/freescale/imx8qm-mek.dts
@@ -670,7 +670,7 @@ &i2c0_mipi0 {
 	#size-cells = <0>;
         pinctrl-names = "default";
         pinctrl-0 = <&pinctrl_mipi0_lpi2c0>;
-	clock-frequency = <100000>;
+	clock-frequency = <1000000>;
 	status = "disabled";
 
 	adv_bridge0: adv7535@3d {
@@ -952,7 +952,6 @@ &fec1 {
 	fsl,magic-packet;
 	nvmem-cells = <&fec_mac0>;
 	nvmem-cell-names = "mac-address";
-	rx-internal-delay-ps = <2000>;
 	status = "okay";
 
 	mdio {
@@ -1094,7 +1093,7 @@ &usdhc2 {
 &i2c0 {
 	#address-cells = <1>;
 	#size-cells = <0>;
-	clock-frequency = <100000>; // 1M for NCB
+	clock-frequency = <1000000>;
 	pinctrl-names = "default";
 	pinctrl-0 = <&pinctrl_i2c0>;
 	status = "okay";
@@ -1129,7 +1128,7 @@ max7322: gpio@68 {
 		reg = <0x68>;
 		gpio-controller;
 		#gpio-cells = <2>;
-		status = "okay"; // disable for NCB
+		status = "disabled";
 	};
 
 	mpl3115@60 {
-- 
2.25.1

